docs/reference/style-transfer.md

```diff
@@ -26,7 +26,7 @@ function modelLoaded() {
   console.log("Model Loaded!");
 }
 // Grab a img element and generate a new image.
-style.transfer(document.getElementById("img"), function(err, resultImg) {
+style.transfer(document.getElementById("img"), function(resultImg, err) {
```

Comment:
This is great! I think our convention is to use `result` and `error`, and then the image itself would be in `result.image`. Is that not the case with `ml5.StyleTransfer()`?

Reply:
I think this example has `(err, result)` https://editor.p5js.org/ml5/sketches/WYjvcqYU6vN ?

Reply:
Yes, thanks! Checked the example and the documentation is updated. I prefer `result` and `error` as the names in the callback for ml5 examples. There's a little inconsistency here where it should probably be `result.image` instead of `result.src` for the actual image data? @yining1023 is there a reason why it is `src`?

Reply:
@shiffman - just picking up this thread after the winter break hiatus. 

The style transfer model returns an HTMLImageElement which is why we return `result.src`. since we're getting back the base64 encoded image. We can update the return results in style transfer, but for our current API, this should be correct. 

```js
// Converts a tf to DOM img
const array3DToImage = (tensor) => {
  const [imgHeight, imgWidth] = tensor.shape;
  const data = tensor.dataSync();
  const canvas = document.createElement('canvas');
  canvas.width = imgWidth;
  canvas.height = imgHeight;
  const ctx = canvas.getContext('2d');
  const imageData = ctx.getImageData(0, 0, canvas.width, canvas.height);

  for (let i = 0; i < imgWidth * imgHeight; i += 1) {
    const j = i * 4;
    const k = i * 3;
    imageData.data[j + 0] = Math.floor(256 * data[k + 0]);
    imageData.data[j + 1] = Math.floor(256 * data[k + 1]);
    imageData.data[j + 2] = Math.floor(256 * data[k + 2]);
    imageData.data[j + 3] = 255;
  }
  ctx.putImageData(imageData, 0, 0);

  // Create img HTML element from canvas
  const dataUrl = canvas.toDataURL();
  const outputImg = document.createElement('img');
  outputImg.src = dataUrl;
  outputImg.style.width = imgWidth;
  outputImg.style.height = imgHeight;
  tensor.dispose();
  return outputImg;
};
```

I'm going to merge this PR in for now and link to this comment in case we want to update  the result results. 

Thanks!